_posts/2020-09-15-open-sourcing-a-private-project.markdown

```diff
@@ -49,13 +49,15 @@ Follow steps on [GitHub](https://docs.github.com/en/enterprise/2.17/user/github/
 
 Follow steps on [GitHub](https://docs.github.com/en/github/getting-started-with-github/create-a-repo) to create a new repository and make sure to choose the public option when choosing the visibility, and don’t include the README.
 ```
-# Choose which commit you would like to include in the new tree(we chose the first commit. Use git log and copy the commit hash.
+# Choose which commit you would like to include in the new tree(we chose the first commit). Use git log and copy the commit hash.
 $ git reset --soft <commit hash>
 $ git commit --amend
-$ adjust commit title as needed
+# adjust commit title as needed
+$ git remote rename origin <oldrepo>
 $ git remote add origin <clone new GitHub repository>
 $ git push origin <main branch>
 ```
+If you are looking to squash the last number of commits into a single one in the commit history, we can use the git reset --soft [commit] option. In our case we squashed all commits against the first one, but deleting the repo and recreating it would have given the same effect. There's an interesting side effect in this approach though, GitHub seems to detect contributions from other developers while if it was a repo from scratch it wouldn't.
```

Comment:
Looks good. 

I was thinking that not all contributors will appear (the ones in the intermediate commits), only the one that did the commit over which we squashed, but maybe its not even worth mentioning it.